generators/app/templates/ci/.circleci/config.yml

```diff
@@ -60,21 +60,21 @@ jobs:
       - store_artifacts:
           path: htmlcov
 
-  black:
+  ruff-format:
     executor: base-python
     steps:
       - get_code_and_cache
       - run:
-          name: run black
-          command: make black
+          name: run ruff-format
+          command: make ruff-format
 
-  ruff:
+  ruff-check:
```

Comment:
Even if the command name is `check`, I think it's clearer to name it `ruff-lint` here

Reply:
(my comments are for all CI configs, of course)

Reply:
done